src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -252,6 +261,12 @@ private void updateIdentity() {
         mail.setText(clientConfiguration.getAccount().getUser());
     }
 
+    private void updateAvatar(Identity identity) {
+        if (identity != null) {
```

Comment:
how can this be null? if identity is null, this code should never be reached (line 235 catches that case)